eyewear_shop/data/qweb_view.xml

```diff
@@ -1,139 +1,484 @@
-                    <div class="page">
-                        <div class="row">
-                            <div class="col h2">
-                                <span>Patient Details</span>
-                            </div>
-                        </div>
-                        <div class="row">
-                            <t t-if="doc.name">
-                                <span t-field="doc.name"/>
-                                <br/>
-                            </t>
-                            <t t-if="doc.street">
-                                <span t-field="doc.street"/>
-                                <br/>
-                            </t>
-                            <t t-if="doc.mobile">
-                                <span t-field="doc.mobile"/>
-                                <br/>
-                            </t>
-                            <t t-if="doc.email">
-                                <span t-field="doc.email"/>
-                                <br/>
-                            </t>
-                        </div>
-                        <div class="row">
-                            <div class="col h2">
-                                <span>Eye Test Results</span>
-                            </div>
-                        </div>
+  <!-- eyetest report -->
+  <record id="studio_report_docume_912bd635-16ab-499e-841c-bf4c99bfd54d" model="ir.ui.view">
+    <field name="name">eyewear_shop.studio_report_docume_912bd635-16ab-499e-841c-bf4c99bfd54d_document</field>
+    <field name="key">eyewear_shop.studio_report_docume_912bd635-16ab-499e-841c-bf4c99bfd54d_document</field>
+    <field name="type">qweb</field>
+    <field name="active" eval="True"/>
+    <field name="arch" type="xml">
+      <t t-name="studio_report_document">
+        <t t-call="web.external_layout">
+          <div class="page">
+            <div class="oe_structure"/>
+          </div>
+        </t>
+      </t>
+    </field>
+  </record>
+  <record id="studio_main_report_9c425e84-4446-43be-acf2-b6b705884663" model="ir.ui.view">
+    <field name="name">eyewear_shop.studio_report_docume_912bd635-16ab-499e-841c-bf4c99bfd54d</field>
+    <field name="key">eyewear_shop.studio_report_docume_912bd635-16ab-499e-841c-bf4c99bfd54d</field>
+    <field name="type">qweb</field>
+    <field name="active" eval="True"/>
+    <field name="arch" type="xml">
+      <t t-name="studio_main_report">
+        <t t-call="web.html_container">
+          <t t-foreach="docs" t-as="doc">
+            <t t-call="eyewear_shop.studio_report_docume_912bd635-16ab-499e-841c-bf4c99bfd54d_document"/>
+          </t>
+        </t>
+      </t>
+    </field>
+  </record>
+  <record id="web_studio_report_ed_0a0f291c-0b17-4d1e-8813-b63a0518701c" model="ir.ui.view">
+    <field name="name">eyewear_shop.studio_report_docume_912bd635-16ab-499e-841c-bf4c99bfd54d_document</field>
+    <field name="inherit_id" ref="studio_report_docume_912bd635-16ab-499e-841c-bf4c99bfd54d"/>
```

Comment:
No need for the history. Just do a clear view without inheritance here

Reply:
Well when I said that, I imagined you would merge both views. Not removing only the `inherit_id` field here, which does not lead to the same result